test/runtests.jl

```diff
@@ -235,4 +238,21 @@ const DST = DataScienceTraits
     carr = categorical([1, 3, 2], ordered=true)
     @test DST.isordered(carr)
   end
+
+  @testset "Distributions" begin
+    @test scitype(Normal()) <: DST.Distributional
+    @test scitype(Exponential()) <: DST.Distributional
```

Comment:
Add `elscitype` tests:
```suggestion
    @test scitype(Normal()) <: DST.Distributional
    @test scitype(Exponential()) <: DST.Distributional
    @test elscitype(fill(Normal(), 3)) <: DST.Distributional
    @test elscitype(fill(Exponential(), 3)) <: DST.Distributional
    @test elscitype([Normal(), missing, Normal()]) <: DST.Distributional
    @test elscitype([Exponential(), missing, Exponential()]) <: DST.Distributional
```